tests/Tests/200__monitor_and_manage/210__alerts/210__alerts.robot

```diff
@@ -284,6 +325,18 @@ Verify "RHODS Probe Success Burn Rate" Alerts Are Not Firing And Continue On Fai
 Skip Test If Alert Is Already Firing
     [Documentation]    Skips tests if ${alert} is already firing
     [Arguments]    ${pm_url}    ${pm_token}    ${rule_group}    ${alert}    ${alert-duration}=${EMPTY}
-    ${alert_is_firing}=    Run Keyword And Return Status    Alert Should Be Firing
+    ${alert_is_firing} =    Run Keyword And Return Status    Alert Should Be Firing
     ...    ${pm_url}    ${pm_token}    ${rule_group}    ${alert}    ${alert-duration}
     Skip If    ${alert_is_firing}    msg=Test skiped because alert "${alert} ${alert-duration}" is already firing
+
+Provoke Tensorflow Image Build Failure
+    [Documentation]   Starts New Build after some time it kill the build and return  name of failed build
+    ${build} =    Search Last Build    namespace=redhat-ods-applications    build-name-includes=tensorflow
+    Deletes Build    namespace=redhat-ods-applications    build_name=${build}
+    ${failed_build_name} =    Starts New Build    namespace=redhat-ods-applications
+    ...    buildconfig=s2i-tensorflow-gpu-cuda-11.4.2-notebook
+    Sleep    60s
```

Comment:
is there a reason for sleeping 1 minute?

Reply:
Because when Start new build, first it takes place in `pending` state and then move to `running`
and also in the next step, we have to kill the pod so we are waiting for 1 min to pod place in running

Reply:
It's better that you use here _Wait Until Build Status Is_  (with a timeout of 15 or 20 mins), as builds can be in Pending state for a long time if cluster resources are scarce

Reply:
Ok
so I Have to pass `Running` state  ${expected-status} to `wait Until build state is`